redbadger/count-dracula
# Create a Google Storage bucket

Closes #30 

## How to test it

-  [x] Create a project on GCP and get projectid:
Go to Google Cloud and check the [project id](https://console.cloud.google.com/home/dashboard?project=count-dracula-production)
- [x] Create a GCP service account
Go to Google cloud IAM console and check  if a service account with the name [`terrafrom-ci`](https://console.cloud.google.com/home/dashboard?project=count-dracula-production) exists
- [x] Encode GCP service account base64
- [x] Store as GitHub secret GOOGLE_APPLICATION_CREDENTIALS the base64 secret
Go to this repo [secrets](https://github.com/redbadger/count-dracula/settings/secrets/actions) and check if  a `GOOGLE_APPLICATION_CREDENTIALS` exists 
-  [x] Create a bucket
Go to Google Cloud and check if a storage bucket exists with the name of [`count-dracula-continous-compliance-prod`](https://console.cloud.google.com/storage/browser/count-dracula-continous-compliance-prod)

![](https://media.giphy.com/media/3oKIPAQeOIj3pOYBt6/giphy.gif)
